case-lib/lib.sh

```diff
@@ -374,7 +374,7 @@ func_mtrace_collect()
     dlogi "Starting ${mtraceCmd[*]} >& $clogfile &"
     # Cleaned up by func_exit_handler() in hijack.sh
     # shellcheck disable=SC2024
-    sudo "${mtraceCmd[@]}" >& "$clogfile" &
+    sudo bash -c "${mtraceCmd[@]} >& $clogfile &"
```

Comment:
The shellcheck warning is valid here: using `@` does not make sense anymore if you include as part of a bigger string. Use `*` as done in the `dlogi` above.

https://mywiki.wooledge.org/Quotes#Expand_argument_lists